packages/dds/map/src/directory.ts

```diff
@@ -1189,7 +1192,7 @@ class SubDirectory extends TypedEventEmitter<IDirectoryEvents> implements IDirec
             return;
         }
         this.clearExceptPendingKeys();
-        this.directory.emit("clear", local, op, this.directory);
+        this.directory.emit("clear", local, message, this.directory);
```

Comment:
Note that this was actually mismatched/wrong before, and was detected by adding typings to `emit`.
Before it was sending `IDirectoryClearOperation` instead of the actual `ISequencedDocumentMessage` as indicated by the typing.